src/mds/Locker.cc

```diff
@@ -2269,30 +2273,64 @@ class C_Locker_RevokeStaleCap : public LockerContext {
   }
 };
 
-int Locker::issue_caps(CInode *in, Capability *only_cap)
+int Locker::get_allowed_caps(CInode *in, Capability *cap,
+                             int &all_allowed, int &loner_allowed,
+                             int &xlocker_allowed)
```

Comment:
I know the Ceph codebase is inconsistent about this for historical reasons, but I believe the style guide says references should be const, and out parameters should all be pointers. We do try and stick to that convention for new and updated code.

Reply:
Sure. will update it.